### PR TITLE
Adds Shock Collars to the game

### PR DESCRIPTION
**Pull request by Bajiru-no-Dansei**
cl Bajiru
add: Added shock collars
/cl

[why]: # Smaller alternative for electropack. This is an invaluable prisoner management tool.

### CODE REVIEW

File: code/game/objects/items/devices/radio/electropack.dm [hunk 1]
```diff
@@ -145,3 +145,13 @@ Code:
 	user << browse(dat, "window=radio")
 	onclose(user, "radio")
 	return
+
+/obj/item/electropack/collar
+	name = "shock collar"
+	desc = "It's for pets."
+	icon = 'icons/obj/clothing/neck.dmi'
+	icon_state = "petcollar"
```
Comment (81Denton) [inline code/game/objects/items/devices/radio/electropack.dm]: nice try

**Comment by 81Denton** [comment]
More like "reverts You can no longer equip pet collars on humans #40946"

**Comment by optimumtact** [comment]
This is a straight buff that replaces the electro pack completely and has no justification.

Further more it appears to be designed to incite an argument.

Not having it, come and speak to me in irc if you're actually serious about an item like this.

**Comment by optimumtact** [comment]
Oh, also this appears to be the second PR of this type you have made.

You're heading on a path that will only lead to you being blocked from contributing.

**Comment by epochayur** [comment]
now THIS is fetish content

**Pull request closed without merging**